tests/unitary/jupyter/test_browser.py

```diff
@@ -135,25 +133,32 @@ def test_nest_applied():
 
 def test_browser_sign_typed_data(display_mock, mock_callback, env):
     signature = env.generate_address()
-    mock_callback("signTypedData", signature)
+    mock_callback("eth_signTypedData_v4", signature)
     data = env.signer.sign_typed_data(
         {"name": "My App"}, {"types": []}, {"data": "0x1234"}
     )
     assert data == signature
 
 
+def test_browser_rpc_inject_js(mocked_token, display_mock, mock_callback):
+    BrowserRPC()
```

Comment:
why is this naked construction necessary?

Reply:
It's testing the constructor

Reply:
```suggestion
    _ = BrowserRPC()  # test the constructor
```